services/movie.py

```diff
@@ -6,9 +7,13 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
Consider using `Optional[list[int]]` from the `typing` module for the `genres_ids` and `actors_ids` parameters to indicate that these lists can be `None`.